source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -151,12 +145,8 @@ void BioXASImagingAppController::createComponentsPane()
 	mw_->addPane(paneWidget, componentPaneCategoryName_, "Carbon Filter Farm", componentPaneIcon_);
 }
 
-void BioXASImagingAppController::onCurrentScanActionStartedImplementation(AMScanAction *action)
-{
-	Q_UNUSED(action)
-}
-
 void BioXASImagingAppController::onCurrentScanActionFinishedImplementation(AMScanAction *action)
 {
-	Q_UNUSED(action)
+	// perfrom the general actions when scan action finished
+	CLSAppController::onCurrentScanActionFinishedImplementation(action);
```

Comment:
I'm not sure that Imaging has a user configuration yet. Will executing this method cause a crash?